identity/src/main/java/com/android/identity/GattClient.java

```diff
@@ -176,7 +218,7 @@ public void onServicesDiscovered(@NonNull BluetoothGatt gatt, int status) {
             // for that in our implementation.
             //
             try {
-                if (!gatt.requestMtu(517)) {
+                if (!mSupportL2CAP && !gatt.requestMtu(517)) {
                     reportError(new Error("Error requesting MTU"));
                     return;
                 }
```

Comment:
Are you sure we don't need to bump the MTU if using L2CAP?

Reply:
Done